### DISCUSSION TITLE
Slider bar seems not showing correct on certain projects

### DISCUSSION THREAD

**Discussion by ajz34** [executablebooks/meta#103] [Q&A]
Hi devs!

I'm not sure whether this is a bug/feature-request or my mis-understand to workflow, so report this issue here.

For my first toy implementation of sphinx-book-theme, I have my page published on [readthedocs page](https://pycrawfordprogproj.readthedocs.io/en/latest/Project_01/Project_01.html) 😃 

However the slider (rolling document up & down) seems weird. There're two sliders, one just right to the contents of that page (seems unnecessary), and another is the web-browser's slider. Additionally, that page could be scrolled even deeper after bottom of the page is approached.

I have no idea where this behavior should be categorized to 😅

**Comment by mmcky** [comment]
@ajz34 I can see what you mean on the link. Thanks. Would you mind letting me know what version of `sphinx-book-theme` you have? You can use `pip show` and will output something like: 

```
▶ pip show sphinx-book-theme
Name: sphinx-book-theme
Version: 0.0.35
Summary: Jupyter Book: Create an online book with Jupyter Notebooks
Home-page: https://jupyterbook.org/
Author: Project Jupyter Contributors
Author-email: jupyter@googlegroups.com
License: BSD
Location: /Users/matthewmckay/anaconda3/envs/ebp/lib/python3.7/site-packages
Requires: pydata-sphinx-theme, pyyaml, docutils, beautifulsoup4, click, sphinx, setuptools, libsass
Required-by: jupyter-book
```

**Reply by ajz34** [reply]
@mmcky The `sphinx-book-theme` version is 0.0.35, and `sphinx` version is 2.4.4.
(build log from https://readthedocs.org/projects/pycrawfordprogproj/builds/11705510/)
Seems https://github.com/executablebooks/sphinx-book-theme/issues/186 is also talking on that topic (0.0)

**Reply by mmcky** [reply]
Ahh -- thanks for linking discussion to the `issue`
